enums/enum-functions.ts

```diff
@@ -12,17 +12,6 @@ export function informationSourceTypeToString(source: InformationSourceType, sho
     return source;
 }
 
-export function displayGraphsTypeToString(source: DisplayGraphs) {
-    switch (source) {
-
-        case DisplayGraphs.CONFUSION_MATRIX: return "Confusion Matrix";
-        case DisplayGraphs.INTER_ANNOTATOR: return "Inter Annotator";
-        case DisplayGraphs.LABEL_DISTRIBUTION: return "Label Distribution";
```

Comment:
should the label distribution be removed?

Reply:
removed enum